demo/index.html

```diff
@@ -17,101 +17,73 @@
     <meta name="apple-mobile-web-app-capable" content="yes">
 
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+    <link rel="import" href="../../iron-demo-helpers/demo-pages-shared-styles.html">
+    <link rel="import" href="../../iron-demo-helpers/demo-snippet.html">
     <link rel="import" href="../../paper-styles/demo-pages.html">
     <link rel="import" href="../paper-toggle-button.html">
 
-    <style is="custom-style">
-      paper-toggle-button {
-        margin-bottom: 40px;
-        display: block;
-      }
-
-      paper-toggle-button:last-child {
-        margin-bottom: 0;
+    <style is="custom-style" include="demo-pages-shared-styles">
+      .vertical-section-container {
+        max-width: 600px;
```

Comment:
I put the minimum for don't need scroll in desktop.
As it is with the "max-width" property, the mobile version will look the same with 500, 550 or 600.